### PR TITLE
CO corpus is updated in config & readme file.

### PR DESCRIPTION
**Pull request by dabhicusp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,8 +93,8 @@ See [this ECMWF documentation](https://confluence.ecmwf.int/display/CKB/ERA5%3A+
 
 | name                 | short name | units   | docs                                              | config                               |
 |----------------------|------------|---------|---------------------------------------------------|--------------------------------------|
-| vorticity (relative) | vo         | s^-1    | https://apps.ecmwf.int/codes/grib/param-db?id=138 | [era5_ml_dv.cfg](raw/era5_ml_dv.cfg) |
-| divergence           | d          | s^-1    | https://apps.ecmwf.int/codes/grib/param-db?id=155 | [era5_ml_dv.cfg](raw/era5_ml_dv.cfg) |
+| vorticity (relative) | vo         | s^-1    | https://apps.ecmwf.int/codes/grib/param-db?id=138 | [era5_ml_dve.cfg](raw/era5_ml_dve.cfg) |
+| divergence           | d          | s^-1    | https://apps.ecmwf.int/codes/grib/param-db?id=155 | [era5_ml_dve.cfg](raw/era5_ml_dve.cfg) |
```
Comment (alxmrs) [inline README.md:97]: Good catch.

File: README.md [hunk 2]
```diff
@@ -240,12 +253,166 @@ sl_reanalysis = xr.open_zarr(
...
+type_of_high_vegetation                          | tvh        | ~         | https://codes.ecmwf.int/grib/param-db/?id=74
+type_of_low_vegetation                           | tvl        | ~         | https://codes.ecmwf.int/grib/param-db/?id=29
+u_component_of_wind                              | u          | m s**-1   | https://codes.ecmwf.int/grib/param-db/?id=131
+v_component_of_wind                              | v          | m s**-1   | https://codes.ecmwf.int/grib/param-db/?id=132
+vertical_velocity                                | w          | Pa s**-1  | https://codes.ecmwf.int/grib/param-db/?id=135
+
+```python
+import xarray as xr
+
+ar_full_37_1h = xr.open_zarr(
```
Comment (alxmrs) [inline README.md:310]: nit: let's have this variable just be called `ar` or `ar_ds`.

File: README.md [hunk 3]
```diff
@@ -240,12 +253,166 @@ sl_reanalysis = xr.open_zarr(
...
+    'gs://gcp-public-data-arco-era5/ar/1959-2022-full_37-1h-0p25deg-chunk-1.zarr-v2/'
+)
+```
+### 1959-2022-full_37-6h-0p25deg-chunk-1.zarr-v2/
```
Comment (alxmrs) [inline README.md:314]: I thought we were getting rid of this version of the dataset? Is this derived from the above dataset?

File: README.md [hunk 4]
```diff
@@ -240,12 +253,166 @@ sl_reanalysis = xr.open_zarr(
...
+v_component_of_wind                              | v          | m s**-1   | https://codes.ecmwf.int/grib/param-db/?id=132
+vertical_velocity                                | w          | Pa s**-1  | https://codes.ecmwf.int/grib/param-db/?id=135
+```python
+import xarray as xr
+
+ar_full_37_6h = xr.open_zarr(
+    'gs://gcp-public-data-arco-era5/ar/1959-2022-full_37-6h-0p25deg-chunk-1.zarr-v2/', 
+)
+```
+### 1959-2022-wb13-6h-0p25deg-chunk-1.zarr-v2/
```
Comment (alxmrs) [inline README.md:365]: Let's get rid of this one, too.

File: README.md (hunk 3 above)
Comment (dabhicusp) [reply README.md:314]: As discussed earlier, the plan was to remove this & this(`1959-2022-wb13-6h-0p25deg-chunk-1.zarr-v2`) both dataset in the `AR-v3`. However, it's worth noting that AR-v3 is currently a work in progress, which is why I've retained both datasets for now.

It's important to highlight that the dataset at **`gs://gcp-public-data-arco-era5/ar/1959-2022-full_37-1h-0p25deg-chunk-1.zarr-v2/`** is a comprehensive dataset, containing all pressure levels and time information that are utilized in both the datasets, namely `1959-2022-full_37-6h-0p25deg-chunk-1.zarr-v2/` and `1959-2022-wb13-6h-0p25deg-chunk-1.zarr-v2/` so we can say that this both dataset can be considered as a derived dataset.

File: README.md (hunk 4 above)
Comment (dabhicusp) [reply README.md:365]: will remove this once `AR-v3` completed.

File: README.md (hunk 3 above)
Comment (alxmrs) [reply README.md:314]: That plan matches my understanding. Thank you.

**Pull request merged**